rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -400,6 +407,23 @@ private RMNode buildRMNode(Node node, Client provider) {
         return rmnode;
     }
 
+    public boolean setNodeAvailable(RMNode node) {
+        Node proactiveProgrammingNode = node.getNode();
+        String proactiveProgrammingNodeUrl = proactiveProgrammingNode.getNodeInformation().getURL();
+        Node downNode = downNodes.remove(proactiveProgrammingNodeUrl);
+
+        if (downNode != null) {
+            logger.info("Setting node as available: " + proactiveProgrammingNodeUrl);
```

Comment:
How many log message like this is printed in the Scheduler.log after this change. Does it pollute the logs or is it acceptable ?